portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Jason</h1>");
```

Comment:
This is a week 2 change. Please start a new PR for this.

Reply:
Noted!

Reply:
Addressing: "This script is no longer invoked by index.html, so let's just remove the file."

I believe I still use it to invoke a quote when I press the button. Am I wrong?

Reply:
You're absolutely right! I made the comment after observing that you had removed the original invocation, without catching the fact that you used it in a different button that you added elsewhere. Great job in pushing back...reviewers make mistakes sometimes!